3600/2/commands/check_number.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'redis'
+require './identifiers'
+
+# :nodoc:
+module CheckNumber
+  include Identifiers
+  def start!(*)
+    if registred?
+      respond_with :message, text: 'You have already registered! Tap /checkin to continue'
+    else
+      save_context :register_from_message
+      respond_with :message, text: 'Please, enter your own Rubizza-number'
+    end
+  end
+
+  def register_from_message(*words)
+    respond_with :message, text: register_session(words)
+  end
+
+  private
+
+  def register_session(number = nil, *)
```

Comment:
strange method signature

Reply:
Renamed method and added new signature for it